### ISSUE TITLE
[Accessibility]A11y_VSCode_FindAndReplace_Keyboard: "Find" editable text is not accessible by the keyboard

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52592]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
Keyboard, NVDA[2018.2],MAS2.1.1

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Status Bar and select “Find And Replace " button.
3.Try to navigate find editable text field by the keyboard.
4.Observed that find editable keyboard is not accessible by the keyboard.

**Actual:**
The keyboard focus goes inside find->Match Case only but not goes on "Match Whole Wod, Use Regular Expression".

**Expected:**
Complete find editable text field should be accessible by the keyboard.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
The user will not be able to access the controls.

**MAS Reference:**
MAS2.1.1: [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](url)

**Attachment for Reference:**
[[Accessibility]A11y_VSCode_FindAndReplace_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2123328/Accessibility.A11y_VSCode_FindAndReplace_Keyboard.pptx)
[[Accessibility]A11y_VSCode_FindAndReplace_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2133160/Accessibility.A11y_VSCode_FindAndReplace_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by chrmarti** [comment]
This works for me. I can tab to all the items.

**Comment by isidorn** [comment]
This works perfectly
